src/components/ValueSet/ValueSetForm.tsx

```diff
@@ -161,15 +178,25 @@ function ConceptFields({
               </FormItem>
             )}
           />
-          <Button
-            type="button"
-            variant="outline"
-            size="icon"
-            onClick={() => handleVerify(index)}
-            disabled={lookupMutation.isPending}
-          >
-            <UpdateIcon className="size-4" />
-          </Button>
+          {parentForm.getValues(
+            `compose.${type}.${nestIndex}.concept.${index}.isVerified` as any,
+          ) ? (
+            <span title="Verified">
+              <CareIcon icon="l-check" className="size-4 text-green-500" />
```

Comment:
adjust the size and the positioning (center?) of the icon here. Notice how due to the size difference, the input box moves rightward.

Also adjust the title - valuesets instead of questionnaire.

Reply:
Go ahead and adjust the mobile responsiveness for rules (try to maintain current design for desktops though).